src/sentry/integrations/api/endpoints/organization_coding_agents.py

```diff
@@ -82,3 +172,279 @@ def get(self, request: Request, organization) -> Response:
+                    },
+                )
+                # Continue with other repos instead of failing entirely
+                continue
+
+        return results
+
+    def post(self, request: Request, organization) -> Response:
+        """Launch a coding agent."""
+        if not features.has("organizations:seer-coding-agent-integrations", organization):
+            return Response({"detail": "Feature not available"}, status=404)
+
+        try:
+            # Validate request and get integration
+            validation_result = self._validate_and_get_integration(request, organization)
+            if isinstance(validation_result, Response):
+                return validation_result
+            integration, installation = validation_result
+
+            # Get autofix state
+            run_id = request.data.get("run_id")
+            if run_id is None:
+                return Response({"error": "run_id is required"}, status=400)
+
+            try:
+                run_id = int(run_id)
+            except (ValueError, TypeError):
+                return Response({"error": "Invalid run_id"}, status=400)
+
+            autofix_state = self._get_autofix_state(run_id, organization)
+            if autofix_state is None:
+                return Response({"error": "Autofix state not found"}, status=400)
+
+            # Get and validate trigger_source
+            trigger_source = request.data.get("trigger_source", "solution")
+            if trigger_source not in ["root_cause", "solution"]:
+                return Response(
+                    {"error": "Invalid trigger_source. Must be 'root_cause' or 'solution'"},
+                    status=400,
+                )
+
+            logger.info(
+                "coding_agent.launch_request",
+                extra={
+                    "organization_id": organization.id,
+                    "integration_id": integration.id,
+                    "run_id": run_id,
+                },
+            )
+
+            # Launch agents for all repos
+            results = self._launch_agents_for_repos(
+                installation, autofix_state, run_id, organization, trigger_source
+            )
+
+            if not results:
+                return Response({"error": "No agents were successfully launched"}, status=500)
+
+            logger.info(
+                "coding_agent.launch_success",
+                extra={
+                    "organization_id": organization.id,
+                    "integration_id": integration.id,
+                    "provider": integration.provider,
+                    "run_id": run_id,
+                    "repos_processed": len(results),
+                },
+            )
+
+            metrics.incr("coding_agent.launch", tags={"provider": integration.provider})
+
+            return Response(
+                {
+                    "success": True,
+                }
+            )
+
+        except Exception as e:
+            logger.exception(
```

Comment:
we work at sentry, so we should try and use `sentry.captureException` where possible! (but as above we have generic handler that does this for all endpoints already)